src/Jackett.Common/Definitions/pixelados.yml

```
@@ -97,13 +100,36 @@ search:
   fields:
     category:
       selector: category_id
-    title:
+    types:
+      selector: types
+    megapack:
+      selector: types:contains(MEGAPACK)
+      optional: true
+    mejorada:
+      selector: types:contains(Mejorada)
+      optional: true
+    corregida:
+      selector: types:contains(Corregida)
+      optional: true
+    audio_editado:
+      selector: types:contains(Audio-Editado)
+      optional: true
+    title_name:
       selector: name
       filters:
+      # Replace Spanish season names like T1, MS (MiniSerie), TU (Temporada Unica) for normalized format S1
         - name: re_replace
           args: ["(?i)T(\\d{1,2})\\b", "S$1"]
         - name: re_replace
-          args: ["(\\d{1,2})[×|x](\\d{1,2})", "S$1E$2"]
+          args: ["(\\d{2})[×|x](\\d{2})", "S$1E$2"]
+        - name: re_replace
+          args: ["(\\d{1})[×|x](\\d{2})", "S0$1E$2"]
+        - name: re_replace
+          args: ["(?i)MS\\b", "S1"]
+        - name: re_replace
+          args: ["(?i)TU\\b", "S1"]
+    title:
+      text: "{{ .Result.title_name }} {{ if or .Result.mejorada .Result.corregida .Result.audio_editado }} PROPER {{ else }}{{ end }}"
```

there will be are lots of extra spaces, with either a blank at the end of the title if not types match, and two blanks between title and proper and one at the end.
```suggestion
      text: "{{ .Result.title_name }}{{ if or .Result.mejorada .Result.corregida .Result.audio_editado }} PROPER{{ else }}{{ end }}"
```

---

My fault, I was testing with multiple configurations and finally only use one "PROPER" tag